src/fetchers.ts

```diff
@@ -77,7 +77,6 @@ type HelmFetch = z.infer<typeof HelmFetchSchema>;
 
 const NixpkgsFetchSchema = BaseFetchSchema.extend({
   type: z.literal("nixpkgs"),
-  channel: z.string(),
```

Comment:
Even with the existing redirect based logic, this would only ever work for `nixpkgs-unstable` because of the regexp. We don't really pin based on any other channel so I'm just removing the option altogether.